tests/search/streamingmatcher/app_elastic/books.xml

```diff
@@ -0,0 +1,181 @@
+  <document type="books" id="id:books:books::http://books.yahoo.com/aynrand/Fountainhead">
```

Comment:
Consider simplifying the id by removing "_http://books.yahoo.com/_". This applies for the entire file.